content/posts/dag-van-de-arbeid-looptocht-1-mei-2025/index.nl.md

```diff
@@ -1,11 +1,14 @@
 ---
 title: Dag van de arbeid-parade - 1 Mei 2025
-description: Techwerkers zijn persoonlijk aanwezig bij de Dag van de Arbeid-parade in Amsterdam
+summary: Techwerkers zijn persoonlijk aanwezig bij de Dag van de Arbeid-parade
```

Comment:
Edited with the CMS, but this is good to check what happens there since we can't easily lint it.